roles/kubernetes-apps/cloud_controller/oci/templates/oci-cloud-provider.yml.j2

```diff
@@ -36,14 +36,17 @@ spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       nodeSelector:
-        node-role.kubernetes.io/master: ""
+        node-role.kubernetes.io/control-plane: ""
```

Comment:
this change breake launch this pods, if no node has this labels.

need to change `nodeSelector:` to NodeAffinity with multiple labels

Reply:
or need check kube_version and use new labels only if version greate then 1.20

Reply:
true